DigitalLearningSolutions.Web/Views/Login/ChooseACentre.cshtml

```diff
@@ -15,65 +16,95 @@
 
     <h1 id="page-heading" class="nhsuk-heading-xl">Choose a centre</h1>
     <p class="nhsuk-body-l">
-      You are registered at more than one centre. Please choose the centre you would like to sign in to.
+      Please choose the centre you would like to sign in to.
     </p>
 
-    @* TODO HEEDLS-912: Remove this temporary button *@
-    <a role="button" class="nhsuk-button delete-button" style="font-size: xx-large; height: 100px; width: 100%;"
-       asp-action="ChooseCentre" asp-route-centreId="0">
-      Log into first available temporary
-    </a>
+    @if (Model.CentreUserDetails.Any(centreRow => !centreRow.IsApproved)) {
+      <partial
+        name="Components/InsetText/Default"
+        model='new InsetTextViewModel("You have an unapproved delegate account. You will receive an email at the address you registered with when your registration has been approved.")' />

```

Comment:
Not really sure what's going on here.
This looks like you're using the view component, but not really using it?
`<vc:inset-text text="You have an unapproved delegate account. You will receive an email at the address you registered with when your registration has been approved." css-class="" />`

Reply:
Ah thanks. I didn't know how to do it properly!